mbid_mapping/mapping/mb_artist_metadata_cache.py

```diff
@@ -85,7 +85,7 @@ def create_json_data(self, row):
                     "name": release_group_name,
                     "mbid": release_group_mbid,
                     "artist_credit_name": artist_credit_name,
-                    "artists": release_group_artists
+                    "artists": sorted(release_group_artists, key=lambda x: x.pop("position", float('inf')))
```

Comment:
Why are you using pop() here? Did you want to modify the list or simply dereference it>

Reply:
So, we just need the position data to sort the artists, but we don't actually require it anywhere else. So, I'm popping the data before saving it to the cache.

Reply:
Interesting approach, but if it works...